velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -36,6 +37,19 @@ void logVectors(const std::vector<RowVectorPtr>& vectors) {
   }
 }
 
+bool supportIgnoreNulls(const std::string& name) {
+  // TODO: Allow ignore nulls on aggregation functions as well. This is
```

Comment:
So IGNORE NULLLs in aggregate functions is weird. In my investigation I came to the conclusion that Presto does not error but just ignores IGNORE NULLs for aggregate windows. 
https://github.com/prestodb/presto/issues/21304

Reply:
Hi @aditi-pandit, thank you for pointing me to https://github.com/prestodb/presto/issues/21304! I knew that Presto doesn't throw when using IGNORE NULLs with aggregation function in window operations, but didn't know IGNORE NULLs was completely ignored. I'll update the comment here.